### ISSUE TITLE
[1.7.10] Question (Seems widely Unknown) - Cursed Earth and Slimes

### ISSUE THREAD

**Issue by Phonavis** [rwtema/extrautilities_old_issues#846]
Will cursed earth allow slime spawns at a higher rate in light? Or is there special code to make sure nothing spawns in light, and if so, could a patch be added to allow slimes to spawn at a higher rate in light on cursed earth?

**Comment by sharkck** [comment]
If I'm not mistaken, doesn't cursed-earth burn up and turn into regular dirt in light?  That wouldn't make this possible?

**Comment by Phonavis** [comment]
No, not always, as far as I know it only burns up in sunlight or at light level 14 or so.. 
That's why I'm curious, does it deactivate in light, or do mobs simply stop spawning?

**Comment by Blue64** [comment]
there's also a few Lamps that don't ignite Cursed Earth (Lapis Lamp from Random Things Mod for example)
